src/Nox.Generator/Application/Dto/EntityUpsertDtoGenerator.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Nox.Generator.Common;
+using Nox.Solution;
+using Nox.Types;
+using Nox.Types.Extensions;
+
+namespace Nox.Generator.Application.Dto;
+
+internal class EntityUpsertDtoGenerator : INoxCodeGenerator
+{
+    public NoxGeneratorKind GeneratorKind => NoxGeneratorKind.Domain;
```

Comment:
@rochar It's used NoxGeneratorKind.Domain for all the Dto models. Is it correct? Shouldn't we use NoxGeneratorKind.Application?

Reply:
NoxGeneratorKind.Application